packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -189,6 +191,18 @@ function useProcessMedia({
           return;
         }
 
```

Comment:
@swissspidy Here we have an uploaded attachment and the server would have this data. See https://github.com/GoogleForCreators/web-stories-wp/pull/11454#issuecomment-1172501960

Reply:
For this case of optimizing an existing video, yes, the server would indeed likely have such information already.

We would need to make an HTTP request to fetch this information and then analyze it, falling back to `getFileInfo()`

A few potential issues I see here:

* Increased complexity due to having to maintain two ways for getting file info
* Unnecessary extra HTTP request in case the server does _not_ have the file info
* This server-side info is most likely only available in a WordPress context, not a decoupled context